collectors/VMStatsCollector.py

```diff
@@ -8,12 +8,14 @@ class VMStatsCollector(BaseCollector):
     def __init__(self):
         super().__init__()
         self.vrops_entity_name = 'virtualmachine'
+        self.rubric = os.environ['RUBRIC']
         self.wait_for_inventory_data()
         self.name = self.__class__.__name__
 
     def collect(self):
         gauges = self.generate_gauges('stats', self.name, self.vrops_entity_name,
```

Comment:
please add hint, that rubric is needed for this particular collector only

Reply:
done